### ISSUE TITLE
Selection of margin value

### ISSUE THREAD

**Issue by iszhaoxin** [thunlp/OpenKE#195]
Not about the code, I would like to ask a question about the hyperparameter selection strategy. 
There are different margin values in different models set in your code. 
Could you tell me how do you choose the value?

**Comment by chenweize1998** [comment]
According to the performance on the validation set.
